### ISSUE TITLE
64 Bit Rasberry Pi Release

### ISSUE THREAD

**Issue by Todd-H-Wilkinson** [muquit/mailsend-go#46]
Hi, can the Linux - 64 bit (tgz, debian and rpm) release be used for 64 bit Debian Rasberry Pi or is a dedicated release required for that platform?

Many thanks.

**Comment by muquit** [comment]
No. Linux binary is targeted for 64 bit Intel/amd processors. Pi uses arm processors.
Thanks.
